transmart-core-db/src/main/groovy/org/transmartproject/db/dataquery/highdim/RepeatedEntriesCollectingTabularResult.groovy

```diff
@@ -75,4 +75,17 @@ class RepeatedEntriesCollectingTabularResult<T extends AbstractDataRow> implemen
             (T) resultItem.call(collected)
         }
     }
+
+    // A helper function that's needed in several subclasses of this
+    @CompileStatic
+    static String safeJoin(List<String> items, String sep) {
+        if(items.size() == 0) return null
+        // null if the list contains only nulls
+        for(i in items) {
+            if(i != null) {
```

Comment:
This could be replaced by `if (items.any { it != null })`, right?
See http://docs.groovy-lang.org/latest/html/groovy-jdk/java/lang/Iterable.html#any(groovy.lang.Closure)